superset/assets/src/dashboard/reducers/getInitialState.js

```diff
@@ -18,11 +18,13 @@ import {
 } from '../util/componentTypes';
 
 export default function(bootstrapData) {
+  // ignore editMode flag from server side.
```

Comment:
I don't think we should ignore this here, I think it's better for it simply _not_ to be set on explore view side.

Reply:
old dashboard needs this feature :)
we can remove after transition time.

Reply:
@graceguo-supercat why does it need that? can you not add a chart to a v1 dashboard from explorer, then get dumped in a v1 dashboard in view mode?

Reply:
editMode=true flag is set from explore view
https://github.com/apache/incubator-superset/blob/c18ef8903409c20b8526b9acd12d3ba1c4c3664b/superset/assets/src/explore/components/SaveModal.jsx#L111

i just feel it's too many hassle to bring this dashboard logic (preview? version1 ? version2?) to explore view.

Reply:
all you have to do is delete `{ edit: 'true' }`,  we discussed that we just never want it to equal true.